### PR TITLE
Make avatar Django 4.0 compatible

### PR DESCRIPTION
**Pull request by luigi370**

### CODE REVIEW

**Comment by PetrDlouhy** [comment]
@luigi370 I think, it would be nice to simplify the code and use `path` instead of `re_path` where possible.
Also I think, that the maintainer/url changes in `setup.py` are mistake.

**Comment by Ali-Aref** [comment]
Nice one this should be merged!

**Review by jayvdb — APPROVED** [review]

**Comment by jayvdb** [comment]
ping @grantmcconnaughey

File: setup.py
Comment (grantmcconnaughey) [inline setup.py:55]: These changes will need to be reverted before I can merge this PR.

**Review by grantmcconnaughey — CHANGES_REQUESTED** [review]

File: setup.py
Comment (ntoll) [reply setup.py:55]: Can this not be merged and then the problematic changes to setup.py reverted before pushing a new version to PyPI..?

Comment (jayvdb) [reply setup.py:55]: IMO it is not worth having such confusing commits in the git history.  The changes in this PR can be mostly automatically generated with `django-codemod` and `django-upgrade`.  If @luigi370 doesnt fix this PR, @grantmcconnaughey might be able to force-push the fix to the PR branch, otherwise someone else can create this PR without the offending changes.

Comment (grantmcconnaughey) [reply setup.py:55]: Yep, I'd prefer to not have that change in the commit history. Removing it then squashing the commits down to 1 oughta do the trick.

Comment (johanneswilm) [reply setup.py:55]: @grantmcconnaughey So that means you will continue to maintain this package? As it currently stands, the package is not Django 4.0 compatible. There are even Django 3.2 issues.

**Comment by yatahaze** [comment]
Any update on this? Feels so close to being able to use this in Django 4....

**Comment by pnizet** [comment]
@luigi370 @grantmcconnaughey Any update on this?

**Comment by ntoll** [comment]
Hi,

I took a quick look at @grantmcconnaughey's status, just in case he's no longer active. Through this I found the following tweet:

https://twitter.com/gmcconnaughey/status/1546263970201337857

I think it means we (who? umm... those who are interested I guess) should feel free to assume this repository is orphaned, fork it and step up as new maintainers (something I personally cannot commit to due to my life being very busy). Those who do carry this work forward, please make sure it has a clear name on PyPI if/when you do a new release :+1: :slightly_smiling_face: . Alternatively, perhaps @jazzband is a good way to go..?

More importantly, I'd like to publicly thank Grant for his work on this project. As an open-source maintainer myself I deeply understand the pressure on folks to release code and unwittingly assume unreasonable responsibilities for all the possible aspects of the code they choose to release. Remember folks, we're volunteers deserving of respect both in terms of time and communication. I hope you flourish and find fulfilment in coding whatever fills you with passion @grantmcconnaughey.

Peace.

**Comment by johanneswilm** [comment]
@ntoll I am already maintainingg more packages than what I'd like to, so I'd be absolutely in favor of putting it somewhere else - like @jazzband. So if we as non-maintainers can get it moved there, I'd be in favor. Do you know what the process is and whom to contact?

Note that I already did some additional updated to move to github actions, etc. in my repository. We should probably have that merged in as well.

**Comment by pnizet** [comment]
@ntoll nothing unrespectfull in my message, I, of course, respect the works done. Big Shout out to @grantmcconnaughey ans all the contributors.
The release is so close... let's move it to somewhere else continue the work :+1:

**Comment by johanneswilm** [comment]
I also want to say that I respect the work of @grantmcconnaughey and I sometimes feel like I'm in a similar situation with some packages where it can take months for me to react to PRs and comments. Thanks for all the work so far!

**Comment by jezdez** [comment]
@ntoll Blast from the past, AMIRITE!

**Comment by johanneswilm** [comment]
Django 4.0 support has already been implemented by means of another PR that has been merged already.

**Pull request closed without merging**
